nixos/tests/docker-tools.nix

```
@@ -46,6 +46,18 @@ let
         echo 'runAsRoot has run.'
       '';
     };
+
+  chownTestImage =
+    pkgs.dockerTools.streamLayeredImage {
+      name = "chown-test";
+      tag = "latest";
+      enableFakechroot = true;
+      fakeRootCommands = ''
+        touch testfile
+        chown 12345:12345 testfile
```

```suggestion
        touch /testfile
        chown 12345:12345 /testfile
```

This way it matches the call in the `Entrypoint`.
It depends on `enableFakechroot = true;` but we have that.